### PR TITLE
exposed RNG to RAVEN python...swig

### PR DESCRIPTION
**Pull request by alfoa**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)

Partially address #597 

##### What are the significant changes in functionality due to this change request?

Exposed the RNG from crow into RAVEN. This allows the possibility to create multiple instances of RAVEN RNG.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: framework/utils/randomUtils.py [hunk 1]
```diff
@@ -95,7 +93,7 @@ def randomSeed(value):
     global npStochEnv
     npStochEnv = np.random.RandomState(value)
   else:
-    crowStochEnv.seedRandom(value) #this has a void return, right?
+    crowStochEnv.seed(value) #this has a void return, right?
```
Comment (PaulTalbot-INL) [inline framework/utils/randomUtils.py]: Doesn't need to change, but the `, right?` could be removed from this comment

File: tests/crow/crowTestUtils.py [hunk 2]
```diff
@@ -19,28 +19,14 @@
...
-def find_interpolationND():
-  """ find the crow interpolationND module and return it. """
-  if sys.version_info.major > 2:
-    import crow_modules.interpolationNDpy3
-    return crow_modules.interpolationNDpy3
-  else:
-    import crow_modules.interpolationNDpy2
-    return crow_modules.interpolationNDpy2
+ravenDir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+print("ravenDir",ravenDir)
```
Comment (PaulTalbot-INL) [inline tests/crow/crowTestUtils.py]: Do you want this print left in?  It's probably not bad to have here, just checking.

**Comment by alfoa** [comment]
I know why of the failure...fixing now

File: crow/crow_modules/randomENGpy2.i [hunk 3]
```diff
@@ -0,0 +1,14 @@
+%module randomENGpy2
```
Comment (mandd) [inline crow/crow_modules/randomENGpy2.i]: would it be possible to leave a description here?

File: crow/crow_modules/randomENGpy3.i [hunk 4]
```diff
@@ -0,0 +1,14 @@
+%module randomENGpy3
```
Comment (mandd) [inline crow/crow_modules/randomENGpy3.i]: same as above

File: tests/crow/crowTestUtils.py (hunk 2 above)
Comment (alfoa) [reply tests/crow/crowTestUtils.py]: Yes. let's keep it here for now.

File: crow/crow_modules/randomENGpy2.i (hunk 3 above)
Comment (alfoa) [reply crow/crow_modules/randomENGpy2.i]: I would prefer to create a guide to do what I just did for the RNG. Can you open an issue and assign to me?

**Comment by alfoa** [comment]
addressed comments....

**Comment by PaulTalbot-INL** [comment]
I'm happy with this being merged from my perspective.

**Comment by alfoa** [comment]
@PaulTalbot-INL Diego approved it... you can go on and merge it if you want

**Pull request merged**